src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Update.java

```diff
@@ -45,15 +41,12 @@ public void onCommand(
             public void run() {
                 MsgUtil.sendMessage(sender, ChatColor.YELLOW + "Checking for updates...");
 
-                final UpdateInfomation updateInfomation = Updater.checkUpdate();
-                final String updateVersion = updateInfomation.getVersion();
-
-                if (updateVersion == null) {
-                    MsgUtil.sendMessage(sender, ChatColor.RED + "Failed check the update, connection issue?");
+                if (plugin.getUpdateWatcher() == null) {
+                    MsgUtil.sendMessage(sender, ChatColor.RED + "Updater seems has been disabled");
                     return;
                 }
 
-                if (!Updater.hasUpdate(updateVersion)) {
+                if (!plugin.getUpdateWatcher().getUpdater().isLatest(plugin.getUpdateWatcher().getUpdater().getCurrentRunning())) {
```

Comment:
So why we not using a method isLatest()?

Reply:
that is design to allow switch different Type of release.